### DISCUSSION TITLE
KKT operator

### DISCUSSION THREAD

**Discussion by asxadf** [yalmip/YALMIP#1078] [Problems using YALMIP] [answered]
Hi Dr. Johan,

I noticed that the KKT operator automatically removed duplicated or redundant inequalities.

However, the number of removed inequalities is different for the same problem. For example, it is "Removed 7 duplicated or redundant inequalities" first, but the second time is "Removed 3 duplicated or redundant inequalities".

Could you please share some insights into this with me?

Best regards,
Xianbang

**Comment by johanlofberg** [comment, marked as answer]
I can only see this happen if the model is extremely sensitive numerically, i.e. the elements in constraints differ on the last digit or something like that.

**Reply by asxadf** [reply]
Got it, maybe I should dig the numerical noises in my model. Thank you!

**Reply by asxadf** [reply]
Indeed, some numerical noises with 1e-8 magnitude lead to this.
